tests/async_engine/test_api_server.py

```diff
@@ -7,6 +7,12 @@
 import pytest
 import requests
 
+from tests.nm_utils.utils_skip import should_skip_test_group
+
+if should_skip_test_group(group_name="TEST_ASYNC_ENGINE"):
+    pytest.skip("TEST_ASYNC_ENGINE=0, skipping async engine test group",
```

Comment:
Remember to update the values 0 here and other places after we have switched to use "enable" and "disable".